trunk/src/faust-generated/hardlim.cc

```diff
@@ -125,8 +123,8 @@ void __rt_func Dsp::compute_static(int count, FAUSTFLOAT *input0, FAUSTFLOAT *in
 
 int Dsp::register_par(const ParamReg& reg)
 {
-	reg.registerFloatVar("hardlim.v1","","SON",N_("Rack output limiter left"),&fVbargraph0, 0, 0.0, 1.0, 0, 0);
-	reg.registerFloatVar("hardlim.v2","","SON",N_("Rack output limiter right"),&fVbargraph1, 0, 0.0, 1.0, 0, 0);
+	reg.registerFloatVar("hardlim.left","","SON",N_("Rack output limiter left"),&fVbargraph0, 0, 0.0, 1.0, 0, 0);
```

Comment:
That is a bit problematic, because all output parameter for bar-graphs been in a group start with `.v`
This is used to filter them out when using a controller script for headless use. Otherwise those messages will spam the terminal.
So, it needs to be `.vleft` `.vright` to avoid issues with that.